docs/src/plasma/qtplasmac.adoc

```diff
@@ -2008,10 +2008,10 @@ This method can be invoked by setting the <<qt_pause-at-end,Pause At End>> param
 [[qt_overcut]]
 [underline]*Over cut*
 
-The torch can be turned off at the end of the hole by setting the *motion.digital-out-03* pin with the M-Codes *M62 (Synchronized with Motion)* or *M64 (Immediate)*.
-After turning the torch off it is necessary to allow the torch to be turned on again before beginning the next cut by resetting the *motion.digital-out-03* pin with the M-Codes *M63* or *M65*, this will be done automatically by the QtPlasmaC G-code parser if it reaches an M5 command without seeing a *M63 P3* or *M65 P3*.
+The torch can be turned off at the end of the hole by setting the `motion.digital-out-03` pin with the M-Codes *M62 (Synchronized with Motion)* or *M64 (Immediate)*.
+After turning the torch off it is necessary to allow the torch to be turned on again before beginning the next cut by resetting the `motion.digital-out-03` pin with the M-codes *M63* or *M65*, this will be done automatically by the QtPlasmaC G-code parser if it reaches an M5 command without seeing a `M63 P3` or `M65 P3`.
```

Comment:
```suggestion
The torch can be turned off at the end of the hole by setting the `motion.digital-out-03` pin with the M-Codes <<mcode:m62-m65,*M62 (Synchronized with Motion)* or *M64 (Immediate)*>>.
After turning the torch off it is necessary to allow the torch to be turned on again before beginning the next cut by resetting the `motion.digital-out-03` pin with the M-codes <<mcode:m62-m65,*M63* or *M65*>>, this will be done automatically by the QtPlasmaC G-code parser if it reaches an M5 command without seeing a `M63 P3` or `M65 P3`.
```